src/main/java/core/basesyntax/strategy/handler/OrderHandler.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class OrderHandler implements OperationHandler {
+    private final FruitDao fruitDao;
```

Comment:
use FruitService instead of FruitDao, the same for other handlers

Reply:
fixed